lib/wallaby/browser.ex

```diff
@@ -1015,16 +1013,16 @@ defmodule Wallaby.Browser do
   @doc """
   Accepts all subsequent JavaScript dialogs in the given session.
   """
-  def accept_dialogs(%Session{}=session) do
-    Driver.accept_dialogs(session)
+  def accept_dialogs(%Session{driver: driver}=session) do
```

Comment:
not for this, but we maybe should get credo going - the narrow spaced assignments bother me  and it's mixed throughout the code base :D